experiments/convert_traj.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+"""
+Convert trajectories from `imitation` format to openai/baselines GAIL format.
+"""
+
+from typing import List
+
+import numpy as np
+
+from imitation.util import rollout as ro_util
```

Comment:
Why `from x import y as z`?

[Google style guide](https://google.github.io/styleguide/pyguide.html#224-decision) says:
> Use from x import y as z if two modules named y are to be imported or if y is an inconveniently long name.

I don't think there's a name clash nor is rollout particularly long